srai/loaders/download.py

```diff
@@ -7,33 +7,34 @@
 
 
 def download_file(
-    url: str, fname: str, chunk_size: int = 1024, force_download: bool = True
+    url: str, filename: Path, chunk_size: int = 1024, force_download: bool = True
```

Comment:
We shouldn't force users to use only string or only Path, we can accept an input like this: `Union[str, "os.PathLike[str]"]` and then wrap it with `Path(...)` at the start of the function.